gallery/__init__.py

```diff
@@ -876,13 +965,21 @@ def get_dir_children(dir_id: int) -> Any:
 @app.route("/api/directory/get/<int:dir_id>")
 @auth.oidc_auth('default')
 @gallery_auth
-def display_files(dir_id: int, internal: bool = False, auth_dict: Optional[Dict[str, Any]] = None):
+def display_files(dir_id: int, internal: bool = False, auth_dict: Optional[Dict[str, Any]] = None, pinned = False):
+    """
+        Grab all children of the given directory, sort, and return as list
+
+        dir_id - ID of the directory to find children of
+        internal: If the call is being made internally, return python object instead of converting to JSON
+        auth_dict: Used for checking if user is E-Board or RTP
+        pinned: Only return pinned or unpinned children
```

Comment:
nit: this is ambiguous `if true, return only pinned children, otherwise, return only unpinned children` maybe?